picard/ui/ui_options_genres.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+
+# Automatically generated - don't edit.
+# Use `python setup.py build_ui` to update it.
+
+from PyQt5 import QtCore, QtGui, QtWidgets
```

Comment:
QtGui is unused (reported by Codacy)

Reply:
Since it's autogenerated, i guess we can ignore

Reply:
Yes, w e should ignore this. This is in all files autogenerated from UI. Maybe we can teach codacy to ignore this issue for the affected files.